src/elements/content-sidebar/activity-feed/task-new/TaskCompletionRuleIcon.js

```diff
@@ -4,6 +4,9 @@ import { FormattedMessage } from 'react-intl';
 import { TASK_COMPLETION_RULE_ANY } from '../../../../constants';
 import messages from './messages';
 import Tooltip from '../../../../components/tooltip';
+// $FlowFixMe
```

Comment:
I don't think this is needed.

Reply:
Any idea on how to fix this? Without $FlowFixMe an error is thrown that it cannot resolve module <path>  It has no flow definitions bc LabelPill is a TS file (cc: @alexkrolick)